### ISSUE TITLE
Clarification why <b> and <i> should be avoided

### ISSUE THREAD

**Issue by mkraetke** [w3c/wcag#3424]
The [German Center for Barrier-Free Reading](https://www.dzblesen.de) recommended that we should generally avoid  `<b>` and `<i>` tags with reference to the WCAG. I did not found a text passage that prohibits the use of `<b>` and `<i>`, however I found this section recommending `<em>` and `<stong>`:

https://www.w3.org/WAI/WCAG21/Techniques/html/H49

I wonder whether there exist a part of the WCAG that prohibits the use of `<b>` and `<i>`? 

Screen readers like JAWS, NVDA or MS Narrator ignore emphasis markup by default or treating it the same, regardless whether you use `<em>` or `<i>`. Furthermore, many editors have only buttons for italic and bold so it's not clear to me why `<em>` should add more semantics than `<i>`? 

Regarding the visual appearance, it's not ideal to use a `<em>` as a generic emphasis tag, as this is handled differently in non-Latin script alphabets.  For example, in Hebrew typography, bold is used to express emphasis and there is no concept of italicized text, but the Web browser would render text in `<em>` with italics. From my point of view, the semantic distinction between _italic_ and **bold** is much more precise than between _emphasized_ and **strong** and I see no reason why `<b>` and `<i>` should be less accessible?

**Comment by j9t** [comment]
`b` and `i` are HTML elements legitimate to use and with different semantics than `strong` and `em`. 

The advice not to use `b` and `i`, or to use `strong` or `em` instead, was popular during XHTML times, but in the end misguided. (“In the end” as it  took the current HTML living standard to clarify.)

**Comment by patrickhlauke** [comment]
> I wonder whether there exist a part of the WCAG that prohibits the use of `<b>` and `<i>`?

WCAG itself is (or rather, tries to be) technology-agnostic. You won't find any normative success criteria that explicitly tell you what you can or can't do explicitly. even understanding documents, which are informative/non-normative, nor techniques, which are also not mandatory as such, won't cover every possible scenario.

In general, the SC this relates to the closest is 1.3.1 Info and Relationships, which - to paraphrase - says that you should choose the most semantically appropriate markup and not just rely on visual styling.

Now, while using `strong` and `em` may currently not have any actual practical/discernable difference in output in screen readers, these are nonetheless different in their purpose/definition from `b` and `i` (see https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-i-element and https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-b-element) (so I wouldn't quite say this differentiation was "misguided" and has now been resolved in any way in the living standard, sorry @j9t)

**Comment by j9t** [comment]
@patrickhlauke, misguided: Saying `i` was the same as `em`, and that `b` was the same as `strong`.

They are not the same.

**Comment by patrickhlauke** [comment]
ah sorry @j9t, missed that nuance in your original reply

**Comment by patrickhlauke** [comment]
> many editors have only buttons for italic and bold

that's a shortcoming of the editors

> Regarding the visual appearance

that;s where CSS comes in

**Comment by mkraetke** [comment]
> In general: prefer em and strong over b and i when you're trying to mark up emphasis. for other cases (as mentioned in the living standard definition for b and i), the latter are of course appropriate.

@patrickhlauke OK, but there is a gap in understanding for me: Authors use bold and italic to write their text. When this text is rendered as HTML, bold and italics are no longer sufficient because they lack semantics? Of course I can use something like `<em class="anger">` which provides more meaning than italicized text. But I don't see any point in it if the author didn't write with that tag from the start.

IMHO, the [HTML living standard definition](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-em-element) is far away from providing a clear distinction. Here is an example:

> The em element isn’t a generic „italics“ element. Sometimes, text is intended to stand out from the rest of the paragraph, as if it was in a different mood or voice. For this, the i element is more appropriate.

**Comment by patrickhlauke** [comment]
>  Authors use bold and italic to write their texts

authors also just bump their text up to be bigger when making headings. authors (assuming you mean mainly WYWISYWG authors here, or authors not familiar with HTML semantics) will need to learn to use the most appropriate markup, or given tools that allow them to do that, and training to make them understand what is/isn't kosher to do.

**Comment by alastc** [comment]
> I wonder whether there exist a part of the WCAG that prohibits the use of <b> and <i>?

If there were an instance where `<b>` or `<i>` were used to convey information visually, and that was different from standard bold or italics text, then you could argue it failed 1.31.

In my personal opinion, I don't think regular use of `<b>` or `<i>` to make some text bold or italic would fail 1.3.1.

**Comment by stevefaulkner** [comment]
May be helpful:

- [Representation of style](https://html5accessibility.com/stuff/2023/02/19/representation-of-style/)
- [Screen Readers support for text level HTML semantics](https://www.tpgi.com/screen-readers-support-for-text-level-html-semantics/)
-

**Comment by mkraetke** [comment]
Thank you @stevefaulkner, these are very helpful resources.

**Comment by fstrr** [comment]
Can this issue be closed?

**Comment by mkraetke** [comment]
I don't think the problem is solved because the cause lies in the ambiguity between physical and so-called "semantic" markup. The HTML working group invented the concept of a purely semantic markup that is disconnected from its visual representation, which did not previously exist in the real world and will continue to cause confusion. But I would be happy if the nonsense about “em” and “strong” being more accessible than “i” and “b” wasn’t written into guidelines.
